src/js/fiContainer.jsx

```diff
@@ -0,0 +1,59 @@
+var React = require('react');
+var divisionHeader = require('./divisionHeader.jsx');
+var fiStatus = require('./fiStatus.jsx');
+
+var fiContainer = React.createClass({
+
+  getInitialState: function(){
+    return this.updateDivisions(this.props.institutions);
+  },
+
+  updateDivisions: function(institutions){
+    var state = {
+      divisions: [
+        {text: 'Not Started', institutions: []},
+        {text: 'In Progress', institutions: []},
+        {text: 'Completed', institutions: []}
+      ]
+    };
+
+    institutions.forEach(function(institution){
+      var status = institution.status;
+      var index = 2;
+
+      if(status < 5) index = 1;
+      if(status === 0) index = 0;
+
+      state.divisions[index].institutions.push(institution);
+
+    });
+
+    return state;
+  },
+
+  updateInstitutions: function(institutions){
+    this.setState(this.updateDivisions(institutions));
+  },
+
+  render: function() {
+    return (
+      <div id="fiContainer">
+        {this.state.divisions.map(function(division, i){
+          var header = null;
+          if(division.institutions.length) header = React.createElement(divisionHeader, {text: division.text});
+          return (
+            <div key={i} className="divisionWrapper">
+              {header}
+              {division.institutions.map(function(institution, i){
+                return React.createElement(fiStatus, {key: i, institution: institution});
```

Comment:
Doing it like this works `<FIStatus key={i} institution={institution} />` with a capital letter first. The lower case version doesn't throw an error :frowning: but doesn't render either.

Of course we'd have to update the require as well, `var FIStatus = require('./fiStatus.jsx');`

Unless you want to fix it now, I'm ok merging this the way it is and we can create an issue to come back and apply the JSX way. I'll handle that, will give me a chance to read through the PR in more detail too.